Description: Ignore
 "[cups-deviced] PID * (usb) crashed on signal 11!" and
 "[cups-deviced] PID * (dnssd) stopped with status 1"
 errors in the log during the test suite. This seems to regularly happen
 on Debian/Ubuntu buildds.
Author: Martin Pitt <mpitt@debian.org>
Author: Didier Raboud <odyx@debian.org>
Last-Update: 2012-11-01

--- a/test/run-stp-tests.sh
+++ b/test/run-stp-tests.sh
@@ -823,7 +823,7 @@
 fi
 
 # Error log messages
-count=`$GREP '^E ' /tmp/cups-$user/log/error_log | wc -l | awk '{print $1}'`
+count=`$GREP '^E ' /tmp/cups-$user/log/error_log | grep -v '(usb) crashed on signal 11' | grep -v '(dnssd) stopped with status 1' | wc -l | awk '{print $1}'`
 if test $count != 33; then
 	echo "FAIL: $count error messages, expected 33."
 	$GREP '^E ' /tmp/cups-$user/log/error_log
